content/blog/learning-to-manage-ilo-7-with-restful-interface-tool-ilorest-6-0.md

```diff
@@ -0,0 +1,757 @@
+
+Let’s get started! 🚀
+
+<!--EndFragment-->
+
+**Installing iLORest 6.0 on Linux**
+
+On Linux, iLORest can be installed as an RPM package. If you already have a previous version installed, you can upgrade it using the -Uvh option.
+For a fresh installation, use:
+
+```bash
+rpm -ivh ilorest-6.0.x86_64.rpm
+```
+
+Here is a screenshot of successful installation.
+
+![](/img/rpm_linux.jpg)
+
+**App account creation during iLORest installation**
+
+During the RPM installation of iLORest 6.0, you might notice that the installer prompts for iLO credentials to create an App account. While this step is optional, HPE strongly recommends creating the App account during installation itself.
+Why?
+
+•	The App account provides an additional method for in-band authentication with iLO 7, enhancing security and flexibility.
+
+•	Once created, the App account allows you to log in without needing traditional credentials every time.
+
+If you choose to skip this step, you can always create the App account later using the following command:
+
+```bash
+ilorest appaccount create -u Administrator -p password –self
+```
+
+By leveraging this new authentication method, managing iLO 7 becomes even more seamless. 🚀
+
+**Installing iLORest 6.0 on Windows**
+
+On Windows, iLORest is installed using an MSI package. During installation, a user interface will appear, prompting you to enter iLO credentials for App account creation.
+Just like on Linux, this step is optional but recommended by HPE, as the App account allows for in-band authentication with iLO 7 without requiring traditional credentials.
+Here’s a screenshot of the App account creation dialog box during installation:
+
+![](/img/windows_msi.jpg)
+
+**Installing and using iLORest on different OSes**
+
+On all operating systems like ubuntu and mac, the iLORest tool can be installed effortlessly via PyPI using the following command:
+```bash
+pip install ilorest
+```
+**Creating an App account**
+
+For these OSes, you can create an App account using the following iLORest command:
+```bash
+ilorest appaccount create -u Administrator -p password –self
+```
+**Logging into iLO 7**
+
+Once the App account is created, you can perform an inband login with:
+```bash
+ilorest login
+```
+If you prefer to log in without using an App account, you can opt for credential-based login instead:
+```bash
+ilorest login --no_app_account -u Administrator -p password
+```
+**Summary**
+
+In this guide, I have demonstrated how to install iLORest 6.0 across different operating systems and leverage the new App account login method introduced in iLO 7.
+
+**Call to Action**
+
+Get started today! Download iLORest 6.0, explore its exciting new features, and take full control of iLO 7 with ease.

```

Comment:
* iLOrest
* External link (\<a href="URL" target="_blank">text\</a> to the iLOrest GitHub releases.